lib/src/hitmap.dart

```diff
@@ -10,8 +10,12 @@ import 'package:coverage/src/util.dart';
 
 /// Contains line and function hit information for a single script.
 class HitMap {
+  /// Constructs a HitMap.
+  HitMap([Map<int, int>? _lineHits, this.funcHits, this.funcNames])
+      : lineHits = _lineHits ?? <int, int>{};
```

Comment:
The argument name does not need a leading underscore.

Also you could optionally omit the generic arguments on the map literal.

```suggestion
  HitMap([Map<int, int>? lineHits, this.funcHits, this.funcNames])
      : lineHits = lineHits ?? {};
```